static/templates/settings/bot-settings.handlebars

```diff
@@ -50,6 +51,12 @@
                             <label for="create_bot_short_name" generated="true" class="text-error"></label>
                         </div>
                     </div>
+                    <div class="input-group" id="service_base_url_inputbox">
+                        <label for="create_service_base_url">{{t "Outgoing webhook service base URL" }}</label>
```

Comment:
Should it just be called e.g. "Payload URL"?  That's what GitHub calls their similar feature, and I'm not sure we need the word "base" in here.